CONTRIBUTING.md

```diff
@@ -21,18 +21,163 @@ To get an overview of the project, read the [README](README.md). Here are some r
 
 For more information specific to how a quarto site works, and for examples of what is possible within a static quarto site, see the main [Quarto website](https://quarto.org/).
 
+#### Visual Editor
+
+The Visual editor in RStudio provides a WYSIWYM interface for authoring Quarto documents. Under the hood, prose in Quarto documents (.qmd files) is written in Markdown, a lightweight set of conventions for formatting plain text files. In fact, Quarto uses Pandoc markdown (a slightly extended version of Markdown that Quarto understands), including tables, citations, cross-references, footnotes, divs/spans, definition lists, attributes, raw HTML/TeX, and more as well as support for executing code cells and viewing their output inline. While Markdown is designed to be easy to read and write, it still requires learning new syntax.
```

Comment:
I'm a little nervous that this is a direct copy and paste of Haldey's material. I think we need to either re-write it directly aimed at our analysts or just point people to Hadley's material. If we were to keep it as is, then we'll need to meet the material licence which states that must clearly state where this text was copied from.